{"name": "Создаём задачу через API → удаляем через UI → проверяем удаление → logout", "status": "broken", "statusDetails": {"message": "TypeError: Provider.text() got an unexpected keyword argument 'max_nb_chart'", "trace": "@pytest.fixture\n    def api_created_task():\n        \"\"\"\n        Создаёт задачу через API и возвращает (task_name, task_id).\n        Удаление не требуется — оно будет через UI.\n        \"\"\"\n        faker=Faker()\n        task_api=TaskAPI()\n    \n        task_name=faker.sentence(nb_words=4)\n>       description=faker.text(max_nb_chart=100)\nE       TypeError: Provider.text() got an unexpected keyword argument 'max_nb_chart'\n\n..\\conftest.py:51: TypeError"}, "description": "\nПроверка удаления задачи через UI:\n1. Создаём задачу через API\n2. Входим в ClickUp\n3. Переходим на доску\n4. Удаляем задачу через интерфейс\n5. Проверяем, что задача исчезла\n6. Выходим из аккаунта\n", "start": 1749918190865, "stop": 1749918190865, "uuid": "70ae2d62-7d4f-47e5-a3cc-08fa232a8188", "historyId": "70fc28dab1ca7a3fc4414a3b10b6ed67", "testCaseId": "70fc28dab1ca7a3fc4414a3b10b6ed67", "fullName": "tests.ui.test_board_page#test_delete_task_ui", "labels": [{"name": "feature", "value": "UI: Удаление задачи"}, {"name": "parentSuite", "value": "tests.ui"}, {"name": "suite", "value": "test_board_page"}, {"name": "host", "value": "A1"}, {"name": "thread", "value": "47600-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.ui.test_board_page"}]}